### DISCUSSION TITLE
000CB09B-CB61-4303-AC0C-B0C5BECE6B9E

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#99] [General]
# (Star Wars Galaxy)

A long time ago in a galaxy far, far away. This add-on provides a catalog of stars, planets and clusters known to exist in the Star Wars universe, both Legends and Disney Canon alike! The galaxy itself is positioned about 923 million light-years away from Earth in the constellation Pegasus

https://celestia.mobi/addon?item=000CB09B-CB61-4303-AC0C-B0C5BECE6B9E

**Comment by DaveBowman2001** [comment]
**NOTE TO ALL CELESTIA USERS** (230814):
A proper "Star Wars Galaxy" addon is currently in active development, which will one day replace this addon once it is ready.


**EDIT (240112)**
The first version of the Star Wars Galaxy add-on is now released and successfully replaced the old _Tatooine, Endor and Death Star_ add-on! Must download first in order for other Star Wars add-ons to work on Celestia. This will be updated periodically for additional planets and stars in the catalogs

**Comment by luca895** [comment]
It isn't working for me

**Reply by DaveBowman2001** [reply]
Can you provide details on how is it not working for you?

**Reply by exocritic5457** [reply]
The Galaxy just doesn't appear for me, there's simply nothing like what Psyche had. I think that's what I get for relying on science

**Reply by luca895** [reply]
When  I point and qo there gets empty black

**Reply by DaveBowman2001** [reply]
Some discussion on Discord just a few hours ago revealed it must be caused by a recently introduced bug that prevents very distant objects from being rendered at all, including (real) galaxies. It is now reported on Celestia's [GitHub](https://github.com/CelestiaProject/Celestia/issues/2085), hopefully a fix will be implemented soon.

**Reply by DaveBowman2001** [reply]
**UPDATE (240207)**: Rendering issue has now been [fixed](https://github.com/CelestiaProject/Celestia/pull/2086)! But we must all patiently wait for the next Celestia update (hopefully on mobile 1.6.9 or 1.7.0) for the fixes to be successfully implemented

**Reply by DaveBowman2001** [reply]
The Galaxy (and by extension, almost all Star Wars addons) should now be working again with the mobile 1.6.9 update just recently released on PlayStore today

**Reply by jasoncao2017-cloud** [reply]
> It isn't working for me

You Just Need to reset The game

**Comment by Marcos270578** [comment]
Good afternoon, I came here to say that this Addon is not working, I have been trying to install it for about 2 years and it does not work, please resolve this

**Reply by DaveBowman2001** [reply]
The Star Wars Galaxy addon does not exist 2 years ago. You might still have installed its older incarnation named _Tatooine, Endor and Death Star_ add-on (which was located in a fictional star within the Milky Way) or using an older version of Celestia that cannot support rendering stars beyond 10 million ly from Earth. You must reinstall this add-on again to see the changes

**Comment by Melneifert** [comment]
The Star Wars Galaxy is working, but when I look at it from certain angles the entire screen becomes extremely bright to where I can barely see anything at all. Also, Felucia should be a lush greenish planet with oceans. Instead, it looks like a bare and desolate moon.

**Reply by DaveBowman2001** [reply]
The flashing was caused by **Cosm's Well**, apparently Celestia has trouble rendering globular clusters at very large distances from Earth. Will post an minor update removing it temporarily from the catalog. On the other hand, **Felucia** (and by extension all other planets) use default exoplanet textures as no one has made any custom map for it yet

**Reply by Melneifert** [reply]
Thank you for the information.

**Reply by DaveBowman2001** [reply]
Sorry it took a while, the new minor update (240424) fixing the flashing issue has now been uploaded

**Comment by Melneifert** [comment]
Thank you for your quick response. The issue is now affecting other
systems, including Earth and the rest of our solar system. I attached
screenshots to this email, so you can see what I’m talking about.
what I mean



Melvin T. Neifert

**Reply by DaveBowman2001** [reply]
I'm sorry. We, add-on devs, don't actually have access to receiving email reports (iirc those are for reporting actual bugs in Celestia's own rendering/code itself, but it is being [retired](https://github.com/celestiamobile/celestia.mobi/issues/91) whilst looking for other solutions). This add-on shouldn't affect the Solar System or any other celestial object beside itself and all Star Wars-related addons....can you elaborate how is it affecting yours? Thanks

**Comment by Melneifert** [comment]
It appears to be our own solar system, but it’s located inside the Star
Wars Galaxy. Makes sense.

Melvin T. Neifert

**Comment by exocritic5457** [comment]
I'd be into seeing addons for systems with multiple inhabited planets like Corellia or Kashyyyk, but it's fine given that this really isn't a big priority at all. Franchise's grown on me.

**Comment by Melneifert** [comment]
I completely understand! Either way, the Celestia app is absolutely
amazing! It’s so much like Space Engine, but you got it all to fit on
mobile devices. That’s one hell of a major accomplishment! I look forward
to seeing more updates.

**Comment by Melneifert** [comment]
I found that there is a blue dwarf star missing from the constellation
Corona Borealis. T CrB (T Corona Borealis, or Blaze Star) is a variable
double star. It is comprised of a red giant and a blue dwarf.

The red giant is visible, but I cannot seem to locate the blue dwarf. The
blue dwarf is a very active star that produces a nova every 80 years. There
have been two recorded novae. One was in 1886, and the second was in 1946,
which changed its visibility from 10+ to 2-3 respectively.

The red giant has an accretion disc that envelopes the blue dwarf until it
produces a nava. The next move event will occur between now and September
and will become as visible as Polaris (magnitude +2) and will be visible
for about a week.

Anyway. I just thought I’d let you know. 😉



Melvin T. Neifert

**Comment by AU-Microscoppi-b** [comment]
This is the biggest waste of time ever but idc, this is every single star in the dddon lol this took me 3 hours
LIST-
(Extragalactic):
1.Kamino

Colonies:
2.Agridorn
3.Alabash
4.Aleron
5.Cato Neimoidia
6.Dankayo
7.Darek
8.Darepp
9.Fondor
10.Foundry
11.Lankashiir
12.Loronar
13.Manwess
14.Mirit
15.Mrisst
16.Vakkar

Core Worlds:
17.Aeton
18.Abregado-rae
19.Alderaan
20.Aldraig
21.Chandrila
22.Chasin
23.Cirius
24.Corellia
25.Corulag
26.Coruscant
27.C'rel
28.Danteel
29.Darada
30.Froz
31.Gandeal
32.Ixtlar
33.Lolnar
34.Orooturoo
35.Osadia
36.Pantolomin

Deep Core:
37.Adana
38.Besero
39.Beshqek
40.Cambria
41.Constancia
42.Crystan
43.Dremulae
44.Dulvoyinn
45.Eclipse
46.Empress Teta
47.Had Abbadon
48.Hakassi
49.Jerrilek
50.Kalist
51.Keeara Major
52.Khomm
53.Prakith
54.Tython
55.Vulpter

Expansion Regions:none lol

Hutt Space:
56.Y'Toub

Inner Rim:
57.Adari
58.Affa
59.Ailon
60.Aldereen
61.Aleen Minor
62.Ambria
63.Aphran
64.Arkam
65.Asrat
66.Atapap
67.Atrig
68.Atzerri
69.Aweris
70.Azurbani
71.Bilbringi
72.Bogden
73.Jakku
74.Pyrshak

Mid Rim:
75.Aargonar
76.Abhean
77.Abraxas
78.Adoris
79.Agriworld-2079
80.Aida
81.Aikhibba
82.Bothawui
83.Chamble
84.Haruun Kal
85.Ingo
86.Jedha
87.Jeyell
88.Kashyyyk
89.Naboo
90.Takodana

Outer Rim:
91.Anoat
92.Bespin
93.Bimmel
94.Birgis
95.Bogano
96.Boonta
97.Crait
98.Dagobah
99.Danadine
100.Dantooine
101.Daslkehnt
102.Dathomir
103.Denab
104.Derilyn
105.D'Qar
106.Dromund
107.Eadu
108.Earth
109.Emmer
110.Endex
111.Endor
112.Erilnar
113.Etti
114.Excarga
115.Felucia
116.Ferro
117.Fest
118.Gabredor
119.Gala
120.Galpos
121.Gand
122.Gandolo
123.Gelda
124.Generis
125.Geonosis
126.Haverling
127.Heffrin
128.Hefi
129.Helska
130.Hoth
131.Ibajji
132.Indu
133.Issor
134.Jabiim
135.Jaemus
136.Jagomir
137.Jaguada
138.Ka'Dedus
139.Karra
140.Kessel
141.Koiogra
142.Kol Huro
143.Korriban
144.Kwevron
145.Lothal
146.Lythos
147.Malachor
148.Mandalore
149.Maridun
150.Marmoth
151.Maryo
152.Minntooine
153.Mon Cala
154.Mustafar
155.Muunilinst
156.Nathema
157.Ninn
158.Noonian
159.Norvan
160.Qella
161.Ryloth
162.Scarif
163.Seline
164.Taris
165.Tatooine
166.Utapau
167.Yavin
168.Zeffo

Unknown Regions:
169.Anch-To
170.Avidich
171.Bogo Rai
172.Carrivar
173.Celwis
174.Cioral
175.Copero
176.Cormit
177.Crakull
178.Crustai
179.Csaus
180.Csilla
181.CZ-198
182.Exegol
183.Ilum
184.Jamiron
185.Kinoss
186.Mobus
187.Naporar
188.Noris
189.Ool
190.O'reen
191.Ornfra
192.Oyokal
193.Paranssos
194.Pesfavri
195.Qoribu
196.Rakata
197.Rentor
198.Rhigar
199.Sarvchi
200.Schesa
201.Sharb
202.Shihon
203.Sposia
204.Ssi'kaat
205.Thearterra
206.Thargo
207.Umaren'k
208.Vagar Praxut
209.Yashuvu
210.Yoggoy
211.Zakuul
212.Zonama Sekot

Wild Space:none lol

Others:

Colonial Station:
213.Colonial Station Cam'co
214.Colonial Station Chaf

Tion Cluster:
215.Derellium
216.Desargorr
217.Eredenn
218.Jaminere
219.Livien

Hapes Cluster:
220-328:Hapes 1-109
329:Algnadesh
330:Arabanth
331:Archais

**Reply by DaveBowman2001** [reply]
Not sure though if listing individual (currently) planet-less star systems is important in the context of this addon, but thanks for the effort anyways! Systems with planets are listed in the ReadMe.txt file however, as I plan to make this updated as periodically as possible....it is still way incomplete than I intended to at the moment

**Comment by AU-Microscoppi-b** [comment]
I wasted another hour:
CLUSTERS:

Anarid Cluster
Zenox Cluster
Koornacht Cluster
Ringali Shell
Galactic Center (SW-OC)
Starswarm Cluster
Aramand Cluster
Greater Plooriod Cluster
Rainos Cluster
Zuni Cluster
Si'Klaata Cluster
Gelviddis Cluster
Hapes Cluster
Inner Cluster:Teke Ro Cluster
Shasfath Cluster
Zeemacht Cluster
Bortele Cluster
Baylin Cluster
Bright Jewel Cluster
Corbett Cluster
Lesser Plooriod Cluster
Mektrun Cluster
Rearqu Cluster
Tyus Cluster
Aturi Cluster
Bi-Boran Cluster
Cron Cluster
Greeb-Streebling Cluster
Jaga's Cluster
Maw Cluster:The Maw
Minos Cluster
Pacanth Reach
Skustell Cluster
Thandon Cluster:Thandon Nebula
Tion Cluster
Vallusk Cluster
Vorzyd Cluster
Ssi-ruuk Cluster:Ssi-Ruuvi Cluster
Morellian Commonwealth
Parthovian Cluster
The Redoubt:Chiss Redoubt
Cosm's Well

BLACK HOLES:

Galactic Center (SW-BH)
Asmeru Anomaly
Black Hole of Quintas:Quintas BH
